static/templates/gear_menu.hbs

```diff
@@ -8,6 +8,30 @@
             view is a Bootstrap nav tab, even though we don't show the tab
             anymore --}}
             <li class="invisible" style="display:none;" role="presentation"><a href="#message_feed_container" data-toggle="tab"></a></li>
+            <li class="org-info org-name">{{realm_name}}</li>
+            <li class="org-info org-url">{{realm_uri}}</li>
+            <li class="org-info org-plan hidden-for-spectators">
+                {{#if is_plan_limited }}
+                    <a href="/plans" target="_blank" rel="noopener noreferrer" role="menuitem">Zulip Cloud Free</a>
+                {{else if is_plan_standard}}
+                    <a href="/plans" target="_blank" rel="noopener noreferrer" role="menuitem">Zulip Cloud Standard</a>
+                {{else if is_plan_standard_sponsored_for_free}}
+                    <a href="/plans" target="_blank" rel="noopener noreferrer" role="menuitem">Zulip Cloud Standard,<br />sponsored for free</a>
+                {{else if is_self_hosted}}
+                    Version {{zulip_version}}
+                {{/if}}
+            </li>
+            {{#if (and is_plan_limited is_owner) }}
+            <li class="org-info">
+                <a href="/upgrade" target="_blank" rel="noopener noreferrer" role="menuitem">Upgrade to:<br /> Zulip Cloud Standard</a>
+            </li>
+            {{/if}}
+            {{#if (and is_self_hosted server_needs_upgrade)}}
+            <li class="org-info">
+                <a href="https://zulip.readthedocs.io/en/stable/production/upgrade-or-modify.html" target="_blank" rel="noopener noreferrer" role="menuitem">Upgrade to the<br /> latest release</a>
```

Comment:
Is the `br` here necessary, or could be instead set CSS for how it should line-wrap? 

The `br` hackery will not work if these need to be tagged for translation.

Reply:
Removed `br` and used CSS `white-space`.